### PR TITLE
NEP-1937 Add caption to filterbar table

### PR DESCRIPTION
**Pull request by KamilaKurys**
screen shot 2016-03-09 at 2 40 04 pm

### CODE REVIEW

File: src/components/Table/Table.react.js [hunk 1]
```diff
@@ -26,17 +26,20 @@ export class Table extends React.Component {
...
 
   render() {
     var headings = this.state.columnHeadings;
+    var tableCaption = this.state.tableCaption;
 
     return (
       <div className="panel panel-responsive">
         <div className="table-responsive">
           <table className="table table-hover table-striped">
+            <caption>{tableCaption}</caption>
```
Comment (andrba) [inline src/components/Table/Table.react.js]: I'm wondering if it creates blank space in case `tableCaption` is not set?

Comment (KamilaKurys) [inline src/components/Table/Table.react.js]: @andrba no it is not. 
screen shot 2016-03-10 at 11 28 50 am

Comment (andrba) [inline src/components/Table/Table.react.js]: I think it does. ATM the margins are equal
image

Comment (KamilaKurys) [inline src/components/Table/Table.react.js]: Hmm @andrba you are right. I will have another look on this.

**Comment by andrba** [comment]
+1

**Pull request merged**